### PR TITLE
CIF-819 - Add meaningful log messages to actions

### PR DESCRIPTION
**Pull request by herzog31**
## Description
For DevOps and customer escalations (CSO) it's important that we can leverage data from monitoring (e.g. Splunk). For this purpose, we add meaningful logs for common errors and warnings in our implementation.

## How Has This Been Tested?
Executing integration tests on CircleCI and validated that logs are available in Splunk.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code passes the code style as defined by ESLint.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by herzog31** [comment]
The logs in Splunk look for example like this:
```javascript
{		
  	 action:	 /ccif-qe-jenkins-core/commerce-cif-commercetools-product@ct-CIF819-346/searchProductsService	
  	 activation_id:	 3fc6b99f1bf34edd86b99f1bf37eddf4	
  	 log_message:	 {"name":"cif","hostname":"1e8af7dd3486","pid":1,"level":50,"baseUrl":"/aem-weretail-demo/product-projections/search?expand=productType&limit=25&offset=0&text.en-US=jacket&markMatchingVariants=false&facet=variants.attributes.color.en%20counting%20products&facet=variants.attributes.size.en%20counting%20products&filter=variants.attributes.color.en","method":"GET","err":{"code":400,"statusCode":400,"status":400,"message":"Malformed parameter: filter: Syntax error while parsing 'variants.attributes.color.en'. Unexpected end of input, expected identChar, '.' or ':' (line 1, column 29):\nvariants.attributes.color.en\n                            ^","originalRequest":{"uri":"/aem-weretail-demo/product-projections/search?expand=productType&limit=25&offset=0&text.en-US=jacket&markMatchingVariants=false&facet=variants.attributes.color.en%20counting%20products&facet=variants.attributes.size.en%20counting%20products&filter=variants.attributes.color.en","headers":{"Authorization":["Bearer ********"]},"method":"GET"},"headers":{"server":["nginx"],"date":["Tue, 30 Apr 2019 09:20:00 GMT"],"content-type":["application/json; charset=utf-8"],"content-length":["529"],"server-timing":["projects;dur=13"],"x-correlation-id":["projects-f10098df-373f-40b0-85e8-0426313ec29a"],"x-served-by":["api-gce-db1d958a.dfw.commercetools.co"],"x-served-config":["sphere-projects-ws-1.0"],"access-control-allow-origin":["*"],"access-control-allow-headers":["Accept, Authorization, Content-Type, Origin, User-Agent"],"access-control-allow-methods":["GET, POST, DELETE, OPTIONS"],"access-control-max-age":["299"],"via":["1.1 google"],"alt-svc":["clear"],"connection":["close"]},"body":{"statusCode":400,"message":"Malformed parameter: filter: Syntax error while parsing 'variants.attributes.color.en'. Unexpected end of input, expected identChar, '.' or ':' (line 1, column 29):\nvariants.attributes.color.en\n                            ^","errors":[{"code":"InvalidInput","message":"Malformed parameter: filter: Syntax error while parsing 'variants.attributes.color.en'. Unexpected end of input, expected identChar, '.' or ':' (line 1, column 29):\nvariants.attributes.color.en\n                            ^"}]},"name":"s"},"msg":"Received error for request","time":"2019-04-30T09:20:00.731Z","v":0}	
  	 log_stream:	 stdout	
  	 log_timestamp:	 2019-04-30T09:20:00.731Z	
  	 namespace:	 ccif-qe-jenkins-customer	
}
```

**Comment by mhaack** [comment]
LGTM, maybe we should add some sample queries how to retrieve the logs in Splunk to the readme or since this is internal Adobe infrastructure to the wiki?

File: src/common/package.json [hunk 1]
```diff
@@ -11,17 +11,18 @@
...
-    "http-status-codes": "1.1.6",
     "@commercetools/api-request-builder": "4.0.1",
     "@commercetools/sdk-client": "1.5.1",
     "@commercetools/sdk-middleware-auth": "4.0.1",
     "@commercetools/sdk-middleware-http": "4.0.1",
     "@commercetools/sdk-middleware-logger": "1.0.7",
-    "isomorphic-fetch": "2.2.1",
     "accept-language-parser": "1.5.0",
-    "tar": "4.4.6",
+    "bunyan": "^1.8.12",
```
Comment (cjelger) [inline src/common/package.json]: Not sure if we still want to stick with the `strict` dependency versions

Comment (herzog31) [reply src/common/package.json]: Good catch. I fixed the version to the current one 👍

**Pull request merged**